WikiWatershed/model-my-watershed
# Add tests to modeling app

Add tests to the modeling app and do a quick clean-up of some existing tests.

**To Test:**

- Run `./scripts/testem.sh`.
- Visit http://localhost:7357/test.html. Note that all the tests pass.
- Stop the testem server.
- Run `./scripts/test.sh`. Note that all the tests pass.

Connects to #242.
